python/lsst/summit/extras/logUtils.py

```diff
@@ -61,12 +62,12 @@ class property ``SPECIAL_ZOO_CASES`` to declare a new type of error animal.
     SPECIAL_ZOO_CASES = ['with gufunc signature (n?,k),(k,m?)->(n?,m?)',
                          ]
```

Comment:
Another recommended reformatting:
```
SPECIAL_ZOO_CASES = [
    "with gufunc signature (n?,k),(k,m?)->(n?,m?)",
]
```

Reply:
Or actually, this is short enough for one line, apologies:
```
SPECIAL_ZOO_CASES = ["with gufunc signature (n?,k),(k,m?)->(n?,m?)"]
```

Reply:
Thanks. At some point this package will probably go `black` and that will be that.